src/Forms/Form.jsx

```diff
@@ -2,13 +2,22 @@ import React from "react";
 import PropTypes from "prop-types";
 import { ValidationContext } from "./Validation";
 
-export default function Form({ children, select }) {
+export default function Form({ children, select, action }) {
+  const submit = (e, validation, submitAction) => {
+    if (e && e.preventDefault) {
+      e.preventDefault();
+    }
+
+    if (validation(e)) {
```

Comment:
what happens if validation is undefined?

Reply:
https://jenkins.uvitatech.com/job/UvitaTeam/job/uvita-webapp/job/master/290/console
made a comment there

Reply:
in jenkins?